www/src/states/use-chat-state.ts

```diff
@@ -62,6 +62,43 @@ const createAssistantMessage = (chatId: string): Message => {
   );
 };
 
+// Migrate function, should be removed in the future
```

Comment:
can you add a task on Linear for us to remove this and add a TODO comment with its reference here?